lua/nordbuddy/init.lua

```diff
@@ -0,0 +1,80 @@
+local utils = require('nordbuddy.utils')
+local palette = require('nordbuddy.palette')
+local colors = require('nordbuddy.colors')
+local Color, c, Group = require('colorbuddy').setup()
+local s = require('colorbuddy.style').styles
+local vim = vim
+local M = {}
+
+function create_custom_styles()
```

Comment:
One thing my language server complains about is these global functions since they are lowercase.
![image](https://user-images.githubusercontent.com/42545625/120054613-f18bc700-bffe-11eb-8709-13f8e4121390.png)

I think rather than uppercasing them, we could put them on the `M` variable?

```suggestion
function M:customizations()
```

Reply:
I split out these methods because they did not feel natural to me to have inside the "export" of this plugin. As I mentioned I'm not very familiar with Lua yet. Is there a convention for naming private methods, like an underscore prefix ? Because I'd prefer that :nerd_face:

Reply:
> inside the "export" of this plugin
Yep good call.

We can also make another `table`:
```
local p = {}
```
and define the functions on that?

To be honest, I'm not entirely familiar with the best practices of lua either.

Reply:
Oh! Actually, I believe you just need to prefix this with a `local` and the language server won't complain
```suggestion
local function create_custom_styles()
```

Reply:
As a heads up, this is where I took most of my inspiration from https://github.com/rockerBOO/boo-colorscheme-nvim. Looks like they follow good lua practices 😄 (which is where I saw the `local functions` 😂 )

Reply:
Ah, nice! Fixed :)

Reply:
> As a heads up, this is where I took most of my inspiration from

Ah, I see!

I've mostly messed around with Lua in the Minecraft ComputerCraft mod thingy and some other game mods, so I haven't really been exposed to good practices.

Also, I should have caught those diagnostics, but apparently I had misconfigured my lsp-config and the server was not actually running... oops!